server.js

```diff
@@ -0,0 +1,88 @@
+const express = require('express');
+const { truncate } = require('fs');
+const { createProxyMiddleware } = require('http-proxy-middleware');
+const path = require('path');
+const chalk = require('chalk');
+const port = 8000;
+const bodyParser = require('body-parser');
+
+const app = express();
+app.use(express.static('client'))
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+
+// app.use('/songdata/', createProxyMiddleware({
+//   target: 'http://52.37.102.63:3005/',
+//   changeOrigin: true
+// }));
+
+// app.use('/relatedTracks/', createProxyMiddleware({
+//   target: 'http://3.15.220.99:3001/',
+//   changeOrigin: true
+// }))
+
+// app.use('/artistBio/', createProxyMiddleware({
+//   target: 'http://34.220.154.45:2000/',
+//   changeOrigin: true
+// }))
+
+// app.use('/songDescription/', createProxyMiddleware({
+//   target: 'http://54.191.20.103:2001/',
+//   changeOrigin: true
+// }))
+
+// app.use('/hashtags/', createProxyMiddleware({
+//   target: 'http://18.189.26.97:4001/',
+//   changeOrigin: true
+// }))
+
+// app.use('/comments/', createProxyMiddleware({
+//   target: 'http://52.14.128.124:4000/',
+//   changeOrigin: true
+// }))
+
+// app.use('/users/', createProxyMiddleware({
+//   target: 'http://18.218.58.9:4002/',
+//   changeOrigin: true
+// }))
```

Comment:
I think you can comfortably delete this cruft as I doubt we're going to be going back to the old EC2 instances from FEC to get data.